### PR TITLE
Ignore archived repos, #88

### PR DESCRIPTION
**Pull request by sprankhub**

### CODE REVIEW

**Comment by sprankhub** [comment]
I already manually removed the state `github_branch_protection.repositories["mage-os-website"]`. So as soon as we merge this, we should be good. It just needs approval from @Vinai, @damienwebdev or @DavidLambauer :)

**Review by Vinai — APPROVED** [review]
LGTM

**Pull request merged**
